src/msgflo.cpp

```diff
@@ -404,19 +408,19 @@ shared_ptr<Engine> createEngine(const EngineConfig config) {
             } else {
                 username = up;
             }
-            cout << "username: " << username << endl;
-            cout << "password: " << password << endl;
+            // cout << "username: " << username << endl;
```

Comment:
Can we remove the code instead of commenting out?
Or if the logging is useful in some case, the move it behind the `debug` flag so it can be toggled on/off at runtime?